test/libwrapper/wrapper.cpp

```diff
@@ -5,3 +5,9 @@
 
 void wrapper()
 {}
+
+void other()
+{}
+
+void tld()
+{}
```

Comment:
As of C++11, it's technically valid for a source file to end without a newline, but it's good practice anyway. If you changed the file extension to `.c`, it would be undefined behaviour even in C17 (as per §5.1.1.2).

Reply:
Good point, and good suggestion, thanks.